src/main/java/com/google/api/codegen/viewmodel/StaticLangRpcStubView.java

```diff
@@ -45,6 +46,19 @@
 
   public abstract String parentName();
 
+  public abstract TransportProtocol transportProtocol();
+
+  public String transportProtocolName() {
+    if (transportProtocol() == TransportProtocol.GRPC) {
+      return "gRPC";
+    }
+    return transportProtocol().toString();
```

Comment:
This is used in the templates, right? Maybe we need to translate `HTTP` to `REST`, which is what we're exposing externally....

Reply:
Agree. Changed to REST